source/extensions/filters/http/fault/fault_filter.cc

```diff
@@ -434,6 +434,15 @@ void FaultFilter::onDestroy() {
   }
 }
 
+bool FaultFilter::activeFaults() {
+  // Check if there is a delay timer or response rate limiter still applying
```

Comment:
Does the response rate limiter go away here? Seems like it's coming from config so it should be constant? The comment reads a bit if it can't actually change